apps/question-bank/src/main/java/gov/cdc/nbs/questionbank/pagerules/PageRuleController.java

```diff
@@ -58,25 +65,28 @@ public CreateRuleResponse deletePageRule(@PathVariable Long ruleId) {
     @PutMapping("/{ruleId}")
     @ResponseBody
     public CreateRuleResponse updatePageRule(@PathVariable Long ruleId,
-                                             @RequestBody CreateRuleRequest request,@PathVariable Long page) throws RuleException {
+            @RequestBody CreateRuleRequest request, @PathVariable Long page) throws RuleException {
         Long userId = userDetailsProvider.getCurrentUserDetails().getId();
-        return pageRuleService.updatePageRule(ruleId, request, userId,page);
+        return pageRuleService.updatePageRule(ruleId, request, userId, page);
     }
 
-    @GetMapping("/{ruleId}")
+    @GetMapping("/{ruleId}/view-rule-by-id")
```

Comment:
Issue: similar to the issue above, to retrieve an item from a REST api, a `GET` is sent to `/collection/{the Id}`. So this should remain `/{ruleId}`

There are several REST best practices docs. I don't have a  favorite, but [here](https://restfulapi.net/resource-naming/) is one worth a read